ui/StatusQ/src/wallet/managetokenscontroller.h

```diff
@@ -133,11 +133,16 @@ class ManageTokensController : public QObject, public QQmlParserStatus
     QSettings m_settings;
     SerializedTokenData m_settingsData; // symbol -> {sortOrder, visible, groupId}
     bool hasSettings() const;
+    void loadSettingsData(bool withGroup = false);
```

Comment:
So, the settings are fully loaded at startup; now with any subsequent change in visibiliy/order we need to resync internally so that the values are up-to-date for the sorting/filtering aspect (for the main view)